actions/lib/utils.py

```diff
@@ -0,0 +1,33 @@
+import re
+from github.InputGitAuthor import InputGitAuthor
+from github.GithubObject import NotSet
+
+__all__ = [
+    'prep_github_params_for_file_ops'
+]
+
+
+def create_github_author(s):
+    # expecting string in the format of "FirstName LastName <email@address>"
+    pattern = re.compile(r"^(.*?)\s+<([a-zA-Z0-9_.+-]+?@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+)?>$")
+    match = re.match(pattern, s)
+    if match:
+        name = match.group(1)
+        email = match.group(2)
+        return InputGitAuthor(name=name, email=email)
+    else:
+        return None
+
+
+def prep_github_params_for_file_ops(author, branch, committer):
+    if not branch:
+        branch = NotSet
+    if committer:
```

Comment:
any reason you don't do if/else here?

Reply:
no reason, probably was adding that later and did not pay attention.